labshare-client/src/views/Index.vue

```diff
@@ -1,12 +1,47 @@
+<i18n>
+    {
+    "en":{
+    "0title": "We combine forces for more SARS-CoV-2 tests",
+    "1title": "Together we unleash the clenched Power of Science!",
```

Comment:
Should be "Together we unleash the power of science!". ("clenched" makes it sound a bit juvenile, like it's the setup for an adolescent joke)